src/benchie/results-output.ts

```diff
@@ -23,3 +30,25 @@ export class ResultsOutputText {
     }
   }
 }
+
+export class ResultsOutputJSON {
+  constructor(readonly results: IScenarioResult[], readonly dir: URL) {}
+
+  print() {
+    const grouped = groupResults(this.results);
+    const output = {};
+    for (let [href, results] of grouped.entries()) {
+      const filepath = new URL(href);
+      const filename = filepath.href.replace(this.dir.href, "");
+      for (let r of results) {
+        output[filename] = {
```

Comment:
what if there are two scenarios in one file?

Reply:
IMO, only the last one would be reported.

Reply:
yes didn't account for multiple scenarios in one file. Will change that. I also want nested scenarios but that will be another time